crates/runtime/benches/setup/mod.rs

```diff
@@ -80,7 +84,9 @@ fn build_app(upload_results_dataset: &Option<String>, connector: &str) -> App {
             .with_dataset(make_spark_dataset("samples.tpch.region", "region"))
             .with_dataset(make_spark_dataset("samples.tpch.supplier", "supplier")),
         "s3" => bench_s3::build_app(app_builder),
+        #[cfg(feature = "postgres")]
```

Comment:
should it be added for spark, s3, spice.ai, etc?

Reply:
I think this is mysql and postgres only as they use `build_app` from feature protected file. spark is feature protected by defines datasets creation directly in this file so works fine.
```
#[cfg(feature = "mysql")]
mod bench_mysql;
```

Reply:
Yes, Sergei explains what's happened here